fybrik/README_mysql.md

```diff
@@ -0,0 +1,185 @@
+
+You will need a copy of the Fybrik repository (`git clone https://github.com/fybrik/fybrik.git`). Set the following environment variables: FYBRIK_DIR for the path of the `fybrik` directory, and AIRBYTE_MODULE_DIR for the path of the `airbyte-module` directory.
+
+1. Install Fybrik Prerequisites. Follow the instruction in the Fybrik [Quick Start Guide](https://fybrik.io/dev/get-started/quickstart/). Stop before the "Install control plane" section.
+
+1. Install Fybrik with a built-in catalog, called Katalog, as opposed to the Openmetadata catalog which is installed by default.
+
+    ```bash
+    cd $FYBRIK_DIR
+    helm install fybrik-crd charts/fybrik-crd -n fybrik-system --wait
+    helm install fybrik charts/fybrik --set coordinator.catalog=katalog --set global.tag=master --set global.imagePullPolicy=Always -n fybrik-system --wait
+    ```
+
+1. Install the Airbyte module:
+    ```bash
+    kubectl apply -f $AIRBYTE_MODULE_DIR/module.yaml -n fybrik-system
+    ```
+
+1. Create a new namespace for the application, and set it as default:
+   ```bash
+   kubectl create namespace fybrik-airbyte-sample
+   kubectl config set-context --current --namespace=fybrik-airbyte-sample
+   ```
+
+1. Create a policy to allow access to any asset (we use a policy that does not restrict access nor mandate any transformations):
+   ```bash
+   kubectl -n fybrik-system create configmap sample-policy --from-file=$AIRBYTE_MODULE_DIR/fybrik/sample-policy.rego
+   kubectl -n fybrik-system label configmap sample-policy openpolicyagent.org/policy=rego
+   while [[ $(kubectl get cm sample-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
+   ```
+
+1. Setup and initialize mysql for reading a dataset
+
+    1. Deploy [mysql](https://bitnami.com/stack/mysql/helm) helm chart in `fybrik-airbyte-sample` namespace:
+      ```bash
+      helm repo add bitnami https://charts.bitnami.com/bitnami
+      helm install mysql bitnami/mysql -n fybrik-airbyte-sample
+      ```
+    2. Use the instructions from the helm chart notes to run a pod that is use as a client and connect to the service:
+      ```bash
+      MYSQL_ROOT_PASSWORD=$(kubectl get secret --namespace fybrik-airbyte-sample mysql -o jsonpath="{.data.mysql-root-password}" | base64 -d)
+      kubectl run mysql-client --rm --tty -i --restart='Never' --image  docker.io/bitnami/mysql:8.0.32-debian-11-r0 --namespace fybrik-airbyte-sample --env MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD --command -- bash
+      mysql -h mysql.fybrik-airbyte-sample.svc.cluster.local -uroot -p"$MYSQL_ROOT_PASSWORD"
+      ```
+   
+    3. In mysql client shell insert the following commands to upload fake data into a new database called `fybrik`:
```

Comment:
In **a** mysql client shell **prompt** ...

Reply:
Fixed Thanks